src/components/Text.jsx

```diff
@@ -73,6 +86,18 @@ const Text =
           font-size: 1rem;
         }
       `}
+
+  ${
+  /** @param {TextProps} props */
+  props =>
+      props.mlg &&
```

Comment:
Hmm `xsm` sounds to me like "xtra-small", but what means `mlg`? Like "medium-large"? What about you use something more descriptive, for example "large". I think is better to keep it clear, even if it uses a lot of characters

Reply:
`large` already exists, and also `lg`.
I was checking and only `NotFound` page has this prop. I could change it to `xlarge` which is a bit bigger (from `2.25rem` to `3rem`)
![image](https://user-images.githubusercontent.com/71513679/159276885-c31c41d9-a19d-4228-a4cd-592ce1452221.png)

Reply:
> `large` already exists, and also `lg`. I was checking and only `NotFound` page has this prop. I could change it to `xlarge` which is a bit bigger (from `2.25rem` to `3rem`) 

I think it's good, I think that looks better with 3rem, what do you think? @Israel-Laguan @angelik0828